Life-Expectancy-and-GDP-Starter/life_expectancy_gdp.ipynb

```diff
@@ -141,6 +141,392 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Trend of Life Expectancy Over Time\n",
+    "plt.figure(figsize=(10, 6))\n",
+    "sn.lineplot(x=\"Year\", y=\"Life\", data=gdp_data, marker='o', label='Life Expectancy')\n",
+    "plt.title('Trend of Life Expectancy Over Time')\n",
+    "plt.xlabel('Year')\n",
+    "plt.ylabel('Life Expectancy')\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The trend line is positive and shows that there is a constant growth in life expectancy over the analyzed period, suggesting that there are no significant changes in the countries analyzed.\n",
+    "* The lower band is wider throughout the period, which shows that one or several countries that have very low life expectancies improved after the year 2006."
```

Comment:
"The lower band is wider throughout the period," - I think this sentence can be improved. Perhaps, "The range decreases over time which shows that one or several countries have an overall increase in life expectancies. This is especially shown after the year 2006."